pkg/virt-handler/cache/cache.go

```diff
@@ -395,7 +395,7 @@ func (d *DomainWatcher) handleResync() {
 		}
 		defer client.Close()
 
-		domain, exists, err := client.GetDomain()
+		domain, exists, err := client.GetDomainWithRuntimeInfo()
```

Comment:
will the call to `GetDomain` in listAllKnownDomains in this package suffer from the same issue?  For example, when virt-handler is restarted, we'll sync all the active domains using that `listAllKnownDomains` function, which appears to not retrive runtime info.

If we end up always needing to retrieve runtime info for virt-handler, then we probably don't even need a new GetDomainWithRuntimeInfo api.